src/ui/zcl_abapgit_gui.clas.abap

```diff
@@ -278,12 +278,39 @@ CLASS ZCL_ABAPGIT_GUI IMPLEMENTATION.
 
 
   METHOD on_event.
+    DATA: lv_action(400) TYPE c,
+          lt_postdata    LIKE postdata,
+          lv_match       TYPE string,
+          lv_ofs TYPE i,
+          lv_len TYPE i,
+          lv_temp TYPE string.
+
+    FIELD-SYMBOLS: <postdata> LIKE LINE OF lt_postdata.
+
+    lv_action = action.
+    lt_postdata = postdata.
+    "hack for java webgui support
+    IF action = '__abapgit_dummy__'.
+      LOOP AT lt_postdata ASSIGNING <postdata>.
```

Comment:
Perhaps, it might help some example - what is on input and what is the expected output. Such a documentation can be stored in commit long message or in a comment.